tests/data/tagged_codelist/tag_species.yaml

```
@@ -0,0 +1,7 @@
+- Species:
+  - CO2:
+     definition: CO2
+     unit: tCO2
```

For consistency with the species in [common-definitions](https://github.com/IAMconsortium/common-definitions/blob/e009dfe10b3eb2c8440b114e44ee3e929c0cabae/definitions/variable/emissions/tag_level-1-species.yaml#L4).

Also, if "definition" would not be specified here, the nomenclature package would replace the tag in "definition" in the code item by the name (i.e., also "CO2"), so replacing it with a longer string helps to make sure that it uses the right replacement (the replacement from the "definition", not the item name.)
```suggestion
     definition: carbon dioxide (CO2)
     unit: Mt CO2
```